client/src/components/App.jsx

```diff
@@ -15,58 +15,85 @@ import Navbar from './Navbar.jsx';
 import Friend from './Friend.jsx';
 import StockDetailPage from './StockDetailPage.jsx';
 import Searchbar from './Searchbar.jsx';
+import { sampleState } from '../../../sampleData/sampleState.js';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      user: {
-        first_name: '',
-        last_name: '',
-        username: '',
-        email: '',
-        cashBalance: 0,
-        rank: null,
-        userPortfolio: [
-          // {
-          //   stockName:
-          //   sharesOwned
-          // }
-        ],
-        friends: [
-          // username, username
-        ]
-      },
-      sessionInfo: {}, // need to be populated with passport
-      selectedFriend: {
-        username: '',
-        rank: 0,
-        portfolioValue: 0,
-        selectedFriendPortfolio: [
-        //  {
-        //     stockName:
-        //     sharesOwned
-        //   },
-        //   {}
-        ]
-      },
-      stockSelected: {
-        name: 'Tesla',
-        symbol: 'TSLA',
-        price: 100,
-        data: [
-          // {},{}
-        ]
-      }
-    };
+    this.fetchSelectedStock = this.fetchSelectedStock.bind(this);
+    this.state = sampleState;
```

Comment:
Can you reset the state back to the initial values instead of the sampleState?

Reply:
I agree with you here, and this is not my code. This PR is a mess. Oof. I am really sorry. It's not supposed to be this large and 90% of the changes are not mine. I thought I merged latest correctly before submitting but apparently not.